apps/core/src/migration/service/convert-monitor-to-app-definition.ts

```diff
@@ -316,6 +331,81 @@ const convertProperties = (
   return getStaticProperties(widgetType);
 };
 
+const convertKpiOrTimelineWidget = (
+  monitorWidget: MonitorWidget,
+  monitorTitle: string,
+): DashboardWidget[] => {
+  // For each metric in a monitor widget, create an application widget
+  if (monitorWidget.metrics) {
+    const maxXCoord = monitorWidget.x + monitorWidget.width - 1;
+    const numWidgets = monitorWidget.metrics.length;
+
+    const appWidgets = [];
+    let row = monitorWidget.y;
+    let column = monitorWidget.x;
+
+    let widgetHeight = convertHeight(1);
```

Comment:
nit: a lot of dups of `convertHeight(1)`;

Reply:
true I actually don't need the else statement, removed